src/topology/vector_bundle.lean

```diff
@@ -249,3 +249,138 @@ lemma is_topological_vector_bundle_is_topological_fiber_bundle :
 λ x, ⟨(trivialization_at R F E x).to_bundle_trivialization, mem_base_set_trivialization_at R F E x⟩
 
 end topological_vector_bundle
+
+/-! ### Constructing topological vector bundles -/
+
+variables (B)
+
+/-- Analogous construction of `topological_fiber_bundle_core` for vector bundles. -/
+@[nolint has_inhabited_instance]
+structure topological_vector_bundle_core (ι : Type*) :=
+(base_set          : ι → set B)
+(is_open_base_set  : ∀i, is_open (base_set i))
```

Comment:
```suggestion
(is_open_base_set  : ∀ i, is_open (base_set i))
```
Spaces after `∀`, here and below.

Reply:
I always put spaces but I copied and pasted this from `topological_fiber_bundle_core` so it's half your (I guess) fault haha

Reply:
I agree its's 99% my fault :-) There are a lot of them still to be fixed later on, by the way!

Reply:
Ok now I did it with a regex so all of them should be fixed